lib/themes/themeSelector.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+class ThemeSelector {
```

Comment:
zoinks. gotta clean this up too

Reply:
Instead of checking for the type, pass it into the constructor when it's being called